samples/SharpDx2D/MainWindow.xaml.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using SharpDX.Direct2D1;
+
+namespace SharpDx2D
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private TimeSpan lastRenderTime;
+        private RenderTarget renderTarget = null;
+        private SharpDX.DirectWrite.TextLayout textLayout;
+        private SharpDX.Direct2D1.Brush brush;
+
+        public MainWindow()
+        {
+            InitializeComponent();
+            Loaded += MainWindow_Loaded;
+            SizeChanged += MainWindow_SizeChanged;
+        }
+
+        private void MainWindow_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            Grid g = (Grid)Content;
```

Comment:
would it be better to use MainImage, instead of the parent grid. So people can add things to the page, and still work fine.